### DISCUSSION TITLE
2.4.0

### DISCUSSION THREAD

**Discussion by baimard** [baimard/gestion#350] [Announcements]
## What's Changed
* Build(deps-dev): Bump webpack from 5.75.0 to 5.76.0 by @dependabot in https://github.com/baimard/gestion/pull/340
* Build(deps): Bump yaml and postcss-loader by @dependabot in https://github.com/baimard/gestion/pull/341
* Build(deps): Bump fast-xml-parser and is-svg by @dependabot in https://github.com/baimard/gestion/pull/343
* Build(deps): Bump fast-xml-parser from 4.2.4 to 4.2.5 by @dependabot in https://github.com/baimard/gestion/pull/346
* Dev 2023 by @baimard in https://github.com/baimard/gestion/pull/349

**Full Changelog**: https://github.com/baimard/gestion/compare/2.3.3...2.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/baimard/gestion/releases/tag/2.4.0'>2.4.0</a>.</em>

**Comment by AMN001** [comment]
...and what's happening with NC 27???

**Reply by baimard** [reply]
Nc27 is supported. You can update
